packages/main/src/components/protected/topbar/components/TopNavbar.jsx

```diff
@@ -137,16 +144,30 @@ const TopNavbar = ({ toggleSidebar }) => {
     getOrganizations();
   }, [setOrgId, user.image_url, setUser]);
 
-  useEffect(() => {
-    UpdateInfo();
-  }, [userProfileImage]);
+  // useEffect(() => {
+  //   UpdateInfo();
+  // }, [userProfileImage]);
 
-  const UpdateInfo = () => {
-    getUserInfo().then(res => {
-      setUserProfileImage(res?.user.image_url);
-      setUser(res.user);
-    });
+  // const UpdateInfo = () => {
```

Comment:
Remove commented-out code, remove console.log and console.error, use toast to display the info to the users.